xbmc/pvr/PVRGUIInfo.cpp

```diff
@@ -1246,6 +1246,19 @@ bool CPVRGUIInfo::GetListItemAndPlayerBool(const CFileItem *item, const CGUIInfo
         return true;
       }
       break;
+    case LISTITEM_IS_PREMIERE:
+      if (item->IsEPG())
+      {
+        const std::shared_ptr<CPVREpgInfoTag> epgTag = item->GetEPGInfoTag();
+        const CDateTime firstAired = epgTag->FirstAiredAsUTC();
+        const CDateTime start = epgTag->StartAsUTC();
+        if (firstAired.IsValid() && start.IsValid())
+        {
+          bValue = firstAired == start;
```

Comment:
So it first aired is only a date can this still work?

Reply:
I think it's probably fair to say an episode is new on its airing date regardless of time.

Reply:
Yep, will change that.

Reply:
Fixed.